### PR TITLE
[noissue] Update readme and logo image

### PR DESCRIPTION
**Pull request by daenamkim**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,11 @@
...
 
-# Sunset Supreme
+# MultiBaas for Google Sheets
 
 - [Live Demo Spreadsheet](https://docs.google.com/spreadsheets/d/1AHCYefYNCjU80X1aSs8Ebre85nVtBeu1cVWmXDIz0_0/edit?usp=sharing)
 - [Demo Video](https://youtu.be/YsbzTZ6Cfvc)
-- [Presentation Slides](Sunset%20Supreme%20Spreadsheet%20Blockchain%20Machine.pdf)
 
-## Tools
+## Developments
```
Comment (shoenseiwaso) [inline README.md:8]:
```suggestion
## Developing
```

File: README.md [hunk 2]
```diff
@@ -1,12 +1,11 @@
 ![logo](./logo.png)
 
-# Sunset Supreme
+# MultiBaas for Google Sheets
 
 - [Live Demo Spreadsheet](https://docs.google.com/spreadsheets/d/1AHCYefYNCjU80X1aSs8Ebre85nVtBeu1cVWmXDIz0_0/edit?usp=sharing)
 - [Demo Video](https://youtu.be/YsbzTZ6Cfvc)
-- [Presentation Slides](Sunset%20Supreme%20Spreadsheet%20Blockchain%20Machine.pdf)
```
Comment (shoenseiwaso) [inline README.md:7]: Let's put the slides back in for now until we can update them.

```suggestion
- [Presentation Slides](Sunset%20Supreme%20Spreadsheet%20Blockchain%20Machine.pdf)

```

File: README.md [hunk 3]
```diff
@@ -1,12 +1,11 @@
 ![logo](./logo.png)
 
-# Sunset Supreme
+# MultiBaas for Google Sheets
```
Comment (shoenseiwaso) [inline README.md:4]:
```suggestion

This spreadsheet plugin was originally developed as part of the [ETHGlobal HackMoney hackathon](https://hack.ethglobal.co/showcase/sunset-supreme-rec3QkXTn6lVq3TH0), where it was called _Sunset Supreme Spreadsheet Blockchain Plugin_. We made the following submissions as part of our entry:
```

**Comment by daenamkim** [comment]
@shoenseiwaso If you are ok can you review 3 other PRs as well?

**Pull request merged**